lib/fxpay/index.js

```diff
@@ -56,13 +56,20 @@
 
 
   exports.purchase = function _purchase(productId, onPurchase, opt) {
-    opt = opt || {};
-    opt.maxTries = opt.maxTries || undefined;
-    opt.pollIntervalMs = opt.pollIntervalMs || undefined;
+    opt = utils.defaults(opt, {
```

Comment:
Not sure of this - `utils.defaults()` only replaces items when they're undefined. So this wouldn't really do anything. If the props of opts are undefined already these defaults won't do anything. The previous code would make things undefined if falsey which is a subtle difference (not sure if that was useful in any case).